libs/wingsdk/src/util/fetch.ts

```diff
@@ -0,0 +1,260 @@
+import { Code, InflightClient } from "../core";
```

Comment:
This file should be under `src/http/fetch.ts` inside a class named `Util`:

`src/http/fetch.ts`:

```js
export class Util {
  public async get(...);
  public async fetch(...);
}
```

The `Util` class name is [special] and will automatically turn into a "global namespace" for the submodule.

[special]: https://github.com/winglang/wing/blob/a69af8e0451101770dda4d240d9521c4cc31191a/libs/wingc/src/type_check.rs#L3136